### DISCUSSION TITLE
Login with Facebook and/or Gmail

### DISCUSSION THREAD

**Discussion by TomDoFuture** [payloadcms/payload#1197] [Feature Requests & Ideas]
It is almost a standard that you can login using your FB or Gmail nowadays,

can this be done? If not then is it a high priority for the develoeprs of payload?

**Comment by imphillipzissou** [comment]
Haven't implemented one myself, but it looks like Payload already accepts Passport [strategies](https://payloadcms.com/docs/authentication/config#strategies) of which FB and Google are available options.
